charts/cluster-autoscaler/README.md

```diff
@@ -274,6 +275,23 @@ Each autoscaling group requires an additional `instanceType` and `region` key to
 
 Read [cluster-autoscaler/cloudprovider/hetzner/README.md](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/hetzner/README.md) for further information on the setup without helm.
 
+### Civo
+
+The following parameters are required:
+
+- `cloudProvider=civo`
+- `autoscalingGroups=...`
```

Comment:
I don't see any template enforcement of this requirement (let me know if I'm missing something). Do you think we can add something that will integrate w/ the existing template spaghetti?

Reply:
I implemented this like the hetzner provider, there is also no validation.

But surely I could add an validation like this:
`charts/cluster-autoscaler/templates/_validation.tpl`:
```
{{- if eq .Values.cloudProvider "civo" -}}
  {{- if gt (len .Values.autoscalingGroups) 0 -}}
    When cloudProvider is set to "civo", autoscalingGroups must be provided.
  {{- end -}}
{{- end -}}
```